static/sass/initial/_type.scss

```diff
@@ -20,32 +14,28 @@
 
 
 
-// Defaults
-// --------
+// Hidden Text & Fout
+// ------------------
 
-.is-hidden {
+.is-hidden,
+.wf-loading {
   @include is-hidden;
```

Comment:
Does this hide all text while Typekit is loading? If so, isn't it somewhat of an accessibility issue for people with slow mobile connections?